app/views/studentLoans/StudentLoansQuestionView.scala.html

```diff
@@ -72,7 +72,7 @@
                     content = Text(messages("studentLoansQuestion.checkbox.2")),
                     value = "pgl",
                     hint = Some(Hint(
```

Comment:
If no Hint is needed you can remove the parameter (I think). I think the default value is None, so you don't need to pass anything.